subprojects/dependency-management/dependency-management.gradle.kts

```diff
@@ -136,6 +136,14 @@ tasks.classpathManifest {
     additionalProjects.add(":runtimeApiInfo")
 }
 
+tasks.clean {
+    doFirst {
+        fileTree("$buildDir/tmp/test files").matching {
+            include("**/read-only-cache/**")
+        }.visit { this.file.setWritable(true) }
+    }
+}
```

Comment:
Where do those files get marked as read-only? Can whatever does that do the cleanup instead?

Reply:
A number of tests do that. They all have the necessary clean up code as part of the test. But if the daemon running the test gets killed, then there are leftovers.

This additional handling is here for such reasons. Unless there are leftover files, the folder `read-only-cache` should not even exist in that file hierarchy.

Reply:
Thanks for the explanation. Can you add this in a comment?